### DISCUSSION TITLE
Shabran Kweyu From Kenya Fraudster

### DISCUSSION THREAD

**Discussion by Gomez1996** [hotspotbilling/phpnuxbill#81] [Fraud Report]
I recently entered into a professional agreement with Shabran Kweyu from Speedcomwifi, seeking his services to customize the PHPNuxBill software according to a specific theme. This theme was clearly outlined and demonstrated in a video I provided, ensuring that the requirements for the project were explicitly communicated and understood.

Upon completion of the initial discussions and agreement on the scope of work, I proceeded with the payment, trusting in the reputation and services of Speedcomwifi. The expectation was to receive a version of PHPNuxBill tailored to the specifications outlined in the video.

However, to my dismay, the files I received post-payment were not the customized PHPNuxBill version as agreed upon. Instead, they were the standard, default files of the software, which were not in line with the contractual agreement. Seeking clarification and resolution, I reached out to Mr. Kweyu to discuss this significant discrepancy.

Regrettably, instead of addressing the issue or providing a solution, Mr. Kweyu chose to terminate our communication. He blocked me, leaving me with no direct recourse to address or resolve the matter through dialogue. This abrupt end to our communication not only left the project incomplete but also raised concerns about the professional practices of Shabran Kweyu and Speedcomwifi.

In light of these events and the lack of resolution, I felt it necessary to report this incident to the authorities. The case is currently filed and under investigation at the Central Police Station in Kisumu. This report has been made in the interest of transparency and to caution others who may consider engaging with Shabran Kweyu or Speedcomwifi for similar services. It is always advisable to exercise due diligence and caution in professional dealings, especially when they involve upfront payments and digital services.

https://github.com/hotspotbilling/phpnuxbill/assets/70133137/b7536d84-4339-47fb-8d92-195c9ccf9cf3

![Screenshot_20231212_165521_Chrome](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/6b4cc37f-a335-4ecf-b528-b8d4a1750cb0)
![Screenshot_20231212_170502_Chrome](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/5d56a9cc-4cbe-421a-88a8-fd11e292b04f)
![Screenshot_20231213_003114_WhatsAppBusiness](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/0249a931-6295-4783-ad09-39856e8bcc06)
![Screenshot_20231213_123219_WhatsAppBusiness](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/c5efba08-c1dc-45f5-98b6-3c70ac2715f4)
![Screenshot_20231213_123430_WhatsAppBusiness](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/6bd2f909-c057-4480-a917-fae52bd3e157)
![Screenshot_20231213_123459_WhatsAppBusiness](https://github.com/hotspotbilling/phpnuxbill/assets/70133137/30ec4072-a4da-4380-b899-3453cce5de92)

**Comment by yathuapps** [comment]
Did you resolve the issue?Very sad reading
